CONTRIBUTING.md

```diff
@@ -1,13 +1,17 @@
 # CONTRIBUTION GUIDELINES
 
 ## Before contributing
+
 Welcome to [TheAlgorithms/C-Plus-Plus](https://github.com/TheAlgorithms/C-Plus-Plus)! Before submitting pull requests, please make sure that you have **read the whole guidelines**. If you have any doubts about this contribution guide, please open [an issue](https://github.com/TheAlgorithms/C-Plus-Plus/issues/new/choose) and clearly state your concerns.
 
 ## Contributing
+
 ### Contributor
+
 We are very happy that you consider implementing algorithms and data structures for others! This repository is referred to and used by learners from around the globe. Being one of our contributors, you agree and confirm that:
+
 - You did your own work.
-    - No plagiarism allowed.  Any plagiarized work will not be merged.
```

Comment:
I thought that markdown wanted four space indents.

Reply:
Nope, Visual Studio Code reports an error. It's only 2 if you want to make it look like:

- One indent
  - Two indents

Reply:
Learn something new every day!